pdns/recursordist/docs/changelog/4.5.rst

```diff
@@ -0,0 +1,201 @@
+Changelogs for 4.5.X
+====================
+
+.. changelog::
+  :version: 4.5.0-alpha1
+  :released: 15th of January 2021
+
+  .. change::
+    :tags: Improvements
+    :pullreq: 9699
+    :tickets: 440
+
+    Introduce "Refresh almost expired" a mechanism to keep the record cache warm.
+
+
+  .. change::
+    :tags: Bug Fixes
+    :pullreq: 9917
+
+    Set the Protocol Buffer 'inBytes' field for zero-length messages.
```

Comment:
This is part of the protozero change, just a week late - you can just add 9917 below or leave this one out entirely. Compared to 4.4.0 there is no difference.